docs/guides/Custom-field/custom-field-integration-guide.md

```diff
@@ -0,0 +1,344 @@
+---
+title: "Custom Fields integration"
+slug: "custom-fields-integration"
+excerpt: "Learn how to configure and apply custom fields to B2B orders, items, and addresses using the Custom Fields API."
+hidden: false
+createdAt: "2025-12-19T17:10:00.000Z"
+updatedAt: "2025-12-19T17:10:00.000Z"
+---
+
+The Custom Fields integration enables B2B stores to capture additional business-specific information during checkout. This guide explains how to use the [Custom Fields API](https://developers.vtex.com/docs/api-reference/custom-fields-api) to define, manage, and apply custom fields such as Cost Center, PO Number, or Location to orders.
+
+The integration supports three main operations:
+
+* **[Custom field settings](#create-custom-field-settings):** Define the structure and behavior of custom fields for a contract.
```

Comment:
Default values are missing from this guide. They are a big part of the Custom Fields journey and should be featured as such. I noticed @geisecosta created a guide reviewed by @anabaarbosa and I think that content should be included here. We should edit that to make it fit here whenever default values apply in the integration.

Reply:
We must make it clear that default values make the checkout custom fields management phase optional.